Documentation is not in line with implementation

1. `podman.image.push.registry` as mentioned below doesn't work. The correct parameter is `podman.push.registry`. 
https://github.com/lexemmens/podman-maven-plugin/blob/9de0c881b794276aeaa75aa6c726a2f6e6560e07/docs/modules/ROOT/pages/general-configuration.adoc?plain=1#L35
https://github.com/lexemmens/podman-maven-plugin/blob/9de0c881b794276aeaa75aa6c726a2f6e6560e07/src/main/java/nl/lexemmens/podman/AbstractPodmanMojo.java#L53

1. The minimum maven version requirement is not up to date. It is should be either `3.8.5 or later` or the `maven.version` should be lowered in the pom.xml
https://github.com/lexemmens/podman-maven-plugin/blob/9de0c881b794276aeaa75aa6c726a2f6e6560e07/docs/modules/ROOT/pages/index.adoc?plain=1#L44
https://github.com/lexemmens/podman-maven-plugin/blob/9de0c881b794276aeaa75aa6c726a2f6e6560e07/pom.xml#L41

      ```bash
      # Below error was thrown when build was done with maven version == 3.6.3
      [ERROR] Failed to execute goal nl.lexemmens:podman-maven-plugin:1.17.0:build (podman-build) on project containers: The plugin nl.lexemmens:podman-maven-plugin:1.17.0 requires Maven version 3.8.5 -> [Help 1]
      ```

--- a/docs/modules/ROOT/pages/general-configuration.adoc
+++ b/docs/modules/ROOT/pages/general-configuration.adoc
@@ -32,7 +32,7 @@ When set to `false`, the plugin will not fail when no `Containerfile` is found.
 |The target registry where the images will be pushed to.
 
 **This property is required.**
-|`podman.image.push.registry`
+|`podman.push.registry`
 
 |skipAuth
 |When set to `true`, Podman will not authenticate the current user with any of the configured registries.

--- a/docs/modules/ROOT/pages/index.adoc
+++ b/docs/modules/ROOT/pages/index.adoc
@@ -41,7 +41,7 @@ Saves an image as a `tar.gz` file to the local disk. This goal is not bound to a
 [#requirements]
 :navtitle: Requirements
 
-- Maven 3.6.2 or later
+- Maven 3.8.5 or later
 - Java 8 or later
 - Podman to be installed on the OS where this plugin will run
 - Registry configuration to be present in Maven settings.

